src/Build/project.json

```diff
@@ -14,19 +16,26 @@
     "netstandard1.5": {
       "dependencies": {
         "NETStandard.Library": "1.6.0",
+        "NuGet.Commands": "4.5.0",
+        "NuGet.Protocol": "4.5.0",
         "System.Collections.Immutable": "1.2.0",
         "System.Collections.NonGeneric": "4.0.1",
         "System.Diagnostics.Contracts": "4.0.1",
         "System.Diagnostics.FileVersionInfo": "4.0.0",
         "System.Diagnostics.Process": "4.1.0",
         "System.Diagnostics.TraceSource": "4.0.0",
+        "System.IO": "4.1.0",
```

Comment:
These had to be added explicitly because the reference to NuGet caused some unwanted upgrades.